recipe/meta.yaml

```diff
@@ -34,6 +34,7 @@ requirements:
     - graphite2
   run:
     - cairo
+    - glib
```

Comment:
```suggestion
    - libglib
```

Reply:
A libglib package that doesn't rely on python.
https://github.com/conda-forge/glib-feedstock/blob/master/recipe/meta.yaml#L97